spec/index.html

```diff
@@ -846,7 +858,7 @@ <h2>Annotation Syntax</h2>
         to both quote and assert a <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">triple</a>,
         which provides a convenient shortcut.
         An annotation can be used to both assert a triple,
-        via an implicit or explicit identifier,
+        via an explicit or implicit or explicit identifier,
```

Comment:
I don't think the order of these matter, but there are only two options, `implicit or explicit` and `explicit or implicit`. `explicit or implicit or explicit` has an extra `explicit`.
```suggestion
        via an explicit or implicit identifier,
```